WordPress/src/main/res/layout/reader_post_detail_header_view_new.xml

```diff
@@ -23,13 +23,12 @@
 
     <org.wordpress.android.ui.reader.views.ReaderFollowButton
         android:id="@+id/header_follow_button"
-        style="@style/Reader.Follow.Button.PostDetail"
+        style="@style/Reader.Follow.Button.New"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:visibility="gone"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toTopOf="@id/layout_blog_section"
-        app:layout_constraintBottom_toBottomOf="@id/layout_blog_section"
+        app:layout_constraintTop_toTopOf="parent"
```

Comment:
Not sure about this change. After the update in the button height (https://github.com/wordpress-mobile/WordPress-Android/pull/19246/commits/4823ba2b5eb22fb2d0747e6bde145b79228b7419), I believe this should still be anchored relative to the top and bottom of `layout_blog_section` as it was previously to match the specs.